metricbeat-tests/runner_test.go

```diff
@@ -51,7 +50,12 @@ type MetricbeatTestSuite struct {
 func (mts *MetricbeatTestSuite) setIndexName() {
 	mVersion := strings.ReplaceAll(mts.Version, "-SNAPSHOT", "")
 
-	index := fmt.Sprintf("metricbeat-%s-%s-%s", mVersion, mts.ServiceName, mts.ServiceVersion)
+	var index string
+	if mts.ServiceName != "" {
+		index = fmt.Sprintf("metricbeat-%s-%s-%s", mVersion, mts.ServiceName, mts.ServiceVersion)
```

Comment:
Actually there is no need to append the service if later on we are going to add a random suffix.

Reply:
Well, it's a matter of making index name more self-explanatory.

If index name generation gets more complicated after new use cases, let's simplify it